flow/ProtocolVersions.cmake

```diff
@@ -8,12 +8,12 @@
 # used and should not be changed from 0.
 #                                                          xyzdev
 #                                                          vvvv
-set(FDB_PV_DEFAULT_VERSION                      "0x0FDB00B072000000LL")
-set(FDB_PV_FUTURE_VERSION                       "0x0FDB00B073000000LL")
-set(FDB_PV_MIN_COMPATIBLE_VERSION               "0x0FDB00B071000000LL")
-set(FDB_PV_MIN_INVALID_VERSION                  "0x0FDB00B074000000LL")
-set(FDB_PV_LEFT_MOST_CHECK                      "0x0FDB00B100000000LL")
-set(FDB_PV_LSB_MASK                             "0xF0FFFFLL")
+set(FDB_PV_DEFAULT_VERSION                      "0x0FDB00C071200000LL")
+set(FDB_PV_FUTURE_VERSION                       "0x0FDB00C072000000LL")
+set(FDB_PV_MIN_COMPATIBLE_VERSION               "0x0FDB00B0710A0000LL")
```

Comment:
min compatible version needs to be set to snowflake/release-71.2's protocol version, which is 0x0FDB00C071200000LL.
The current value of this is snowflake/release-7.1's protocol version, which is two major releases ago

Reply:
Done